src/pymatgen/io/vasp/help.py

```diff
@@ -55,7 +55,6 @@ def get_help(cls, tag: str, fmt: str = "text") -> str:
         tag = tag.upper()
         response = requests.get(
             f"https://www.vasp.at/wiki/index.php/{tag}",
-            verify=False,  # noqa: S501
```

Comment:
Need confirm, I can see this helper function was added with SSL certificate turned off (#1791), not sure if this is intended for any reason. The `print_help` method seems to work fine with `verify` enabled:
```python
from pymatgen.io.vasp.help import VaspDoc


doc = VaspDoc()
doc.print_help("ISYM")
```

Reply:
academic websites sometimes lack SSL certificates. perhaps VASP docs did in the past, but since they have SSL now, no reason not to enable